sky/client/cli/command.py

```diff
@@ -6169,19 +6169,22 @@ def api_logs(request_id: Optional[str], server_logs: bool,
                 **_get_shell_complete_args(_complete_api_request))
 @flags.all_option('Cancel all your requests.')
 @flags.all_users_option('Cancel all requests from all users.')
+@flags.yes_option('Skip confirmation (for when -a or -u is specified).')
```

Comment:
```suggestion
@flags.yes_option()
```
let's keep the flag helptext consistent with other instances, which all just use `@flags.yes_option()`.